util/types/json/json_test.go

```diff
@@ -27,19 +28,28 @@ func TestT(t *testing.T) {
 	TestingT(t)
 }
 
+func parseFromStringPanic(s string) JSON {
```

Comment:
`parseFromStringPanic` is not a good name, `mustParseFromString` would confirm to Go idiom.